### PR TITLE
Fixed the Deploy Contract[Starknet/ReactJs]

### PR DESCRIPTION
**Pull request by KodeSage**
## Description

_Concise and detailed description of proposed changes.

## Additional Information

- [x] I have read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/argentlabs/Starknet-Scaffold/pulls)

## Related Issues

_Closes #45 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

### CODE REVIEW

File: burner/src/mock/account.cairo [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #[starknet::contract]
 mod Account {
-
```
Comment (Darlington02) [inline burner/src/mock/account.cairo:3]: what change was made here? if not necessary, could you please undo?

File: frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx [hunk 2]
```diff
@@ -42,6 +51,37 @@ function ScaffoldDeployer() {
...
+  const handleDeploy = async (e: React.FormEvent) => {
+    try {
+      e.preventDefault();
+      if (!classHash) {
+        throw new Error("No class hash");
+      }
+      if (!isConnected || !account) {
+        throw new Error("Connect wallet to continue");
+      }
+      // if(constructorArguments === ("" || null)){
```
Comment (Darlington02) [inline frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx:64]: code looks a bit messy, please clear up comments/unused codes

File: frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx [hunk 3]
```diff
@@ -5,6 +5,10 @@ import trash from "../../../../public/assets/deleteIcon.svg";
 import { useRef, useState } from "react";
 import Header from "../Header";
 import Image from "next/image";
+import { UniversalDeployerContractPayload, provider , CallData} from "starknet";
+import { useAccount,useProvider } from "@starknet-react/core";
```
Comment (Darlington02) [inline frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx:9]: Add a space between useAccount and useProvider

File: frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx [hunk 4]
```diff
@@ -100,24 +140,36 @@ function ScaffoldDeployer() {
             Declare
```
Comment (Darlington02) [inline frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx:140]: work on ensuring codes are aligned properly:)

File: frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx [hunk 5]
```diff
@@ -100,24 +140,36 @@ function ScaffoldDeployer() {
...
             type="text"
             className="mt-4 mb-6 text-black p-3 rounded w-[600px]"
             placeholder="Input Class Hash"
+            onChange={(e) => {
+              setClassHash(e.target.value);
+            }}
+            value={classHash}
           />
-          <input
+          {/* <input
```
Comment (Darlington02) [inline frontend/src/app/components/ScaffoldDeployer/ScaffoldDeployer.tsx:154]: please ensure to uncomment the input boxes for taking in the constructor argument and no. of constructor arguments. deploying with constructor arguments will be addressed in another issue

File: frontend/src/app/services/wallet.service.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+// import {
```
Comment (Darlington02) [inline frontend/src/app/services/wallet.service.ts:1]: Delete this wallet.service.ts file, seems to have no use.

**Review by Darlington02 — COMMENTED** [review]
Hi @KodeSage please address the following comments, and I'd merge your changes

File: burner/src/mock/account.cairo (hunk 1 above)
Comment (KodeSage) [reply burner/src/mock/account.cairo:3]: no not at all o

**Pull request merged**